packages/dht/test/integration/ReplicateData.test.ts

```diff
@@ -109,20 +93,19 @@ describe('Replicate data from node to node in DHT', () => {
             const node = nodesById.get(PeerID.fromValue(contact.getPeerDescriptor().nodeId).toKey())!
             let hasDataMarker = ''
             
-            // @ts-expect-error private field
-            if (node.localDataStore.getEntry(dataKey)) {
+            if (hasData(node)) {
                 hasDataMarker = '<-'
             }
 
             // eslint-disable-next-line max-len
-            logger.info(peerIdFromPeerDescriptor(contact.getPeerDescriptor()) + ' ' + getNodeIdFromPeerDescriptor(node.getLocalPeerDescriptor()) + hasDataMarker)
+            logger.info(getNodeIdFromPeerDescriptor(contact.getPeerDescriptor()) + ' ' + getNodeIdFromPeerDescriptor(node.getLocalPeerDescriptor()) + hasDataMarker)
         })
 
         logger.info(NUM_NODES + ' nodes joining layer0 DHT')
         await Promise.all(
-            nodes.map((node) => {
-                if (keyFromPeerDescriptor(node.getLocalPeerDescriptor()) != '0') {
-                    node.joinDht([entrypointDescriptor])
+            nodes.map(async (node, i) => {
+                if (i !== 0) {
+                    await node.joinDht([entrypointDescriptor])
```

Comment:
Wouldn't 'return' be more appropriate here since there is the promise.all()?

Reply:
`joinDht` doesn't return anything (i.e. just `Promise<void>`), so `await` communicates the intention better